diff --git a/packages/gstech-core/modules/axios.js b/packages/gstech-core/modules/axios.js
index 7f490af64d..a0403bbb3e 100644
--- a/packages/gstech-core/modules/axios.js
+++ b/packages/gstech-core/modules/axios.js
@@ -1,6 +1,7 @@
 // @flow
 import type { Axios } from 'axios';
 
+require('axios-debug-log')
 const _ = require('lodash');
 const axios = require('axios');
 const axiosRetry = require('axios-retry');
